express-notes-api/server.js

```diff
@@ -1,104 +1,105 @@
 import express from 'express';
 import * as fs from 'node:fs/promises';
 
-/* This just makes it easier to create error messages for me.
-   And it looks cleaner when written in the code later on. */
-class Error {
-  constructor(errorMsg) {
-    this.error = errorMsg;
-  }
-}
+const error = (message) => ({ error: message });
 
 const port = 8080;
+/**
+  I am used to having to read and write to different files from my Computational Linguistics work.
+  While this api will only ever read and write to one file, I always found it helpful to not hard code
+  where I am reading and writing in case I ever re-used code in another program or I needed to try a
+  different approach and change where I was recording the outcome of my program. Additionally, I would
+  have many different files to read from, and having one variable at the top that held where I was reading
+  from made it easier since I was often retrieving where I wanted to read from on the command line or in another file,
+  and so I couldn't hard code it and had to pass the info to my read and write functions. So it's just 'muscle memory' from Comp Ling.
+ */
 const readFile = './data.json';
 const writeFile = './data.json';
 const app = express();
 const parser = express.json();
 app.use(parser);
 
-/* An async function for grabbing the data from the json file.
-   Data will be parsed before returning.                    */
-async function getJSON(fileUrl) {
-  try {
-    const contents = await fs.readFile(fileUrl, 'utf8');
-    const parsed = JSON.parse(contents);
-    return parsed;
-  } catch (err) {
-    console.log('Error reading json:', err);
-  }
+/**
+   An async function for grabbing the data from the json file.
```

Comment:
Although this is very nit-picky, JSDoc comments don't usually talk _about_ the function, instead they describe what it does. So they usually start with an action verb.